packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -48,7 +48,7 @@ Requires: which
 # puppet-candlepin enables the pki-core module which needs DNF module support
 # That was introduced in 5.5.20 and 6.15.0
 %if 0%{?rhel} == 8
-Requires:   puppet-agent >= 5.5.20
+Requires:   puppet-agent >= 6.1.0
 Conflicts: (puppet-agent >= 6.0.0 with puppet-agent < 6.15.0)
```

Comment:
Can we fold that into `puppet-agent >= 6.15.0`?

Reply:
I considered that, but I wanted to be consistent with the testing in foreman-installer.

Reply:
The way https://github.com/theforeman/puppet-candlepin/pull/200 is written now, it must be `>= 6.15.0` everywhere. With versions older than that it'll always show an error. That's why I wrote https://github.com/theforeman/foreman-installer/pull/683#discussion_r634208321. So IMHO foreman-installer should align with that.